Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -129,7 +133,11 @@ GameListItem::GameListItem(const std::string& _rFileName)
 		ini.GetIfExists("EmuState", "EmulationIssues", &m_issues);
 	}
 
-	if (!m_pImage.empty())
+	if (IsElfOrDol())
+	{
+		m_Bitmap.LoadFile(StrToWxStr(File::GetThemeDir(SConfig::GetInstance().theme_name)) + "filebanner.png", wxBITMAP_TYPE_PNG);
```

Comment:
Why? DOLs/ELFs should have a separate platform icon, not a special banner. This feels odd from the user perspective and especially the code perspective.